packages/mail-client/src/mail-client.spec.ts

```diff
@@ -0,0 +1,47 @@
+import nodemailer from 'nodemailer';
+import { sendMail } from './mail-client';
+import { MailOptions } from './mail-client-types';
+
+describe('mail client', () => {
+  class MockMailer {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    sendMail() {}
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    close() {}
+  }
```

Comment:
[q] Why is this a class. Wouldn't it be nicer to have an object?
[req] I think you can avoid disabling lint by using `jest.fn()`.

```suggestion
const mailTransport = {
  sendMail: jest.fn(),  close: jest.fn()
};
```